### PR TITLE
Gazell support for nRF5340 network core

### PR DESCRIPTION
**Pull request by leewkb4567**
Support Gazell on nRF5340 network core.

### CODE REVIEW

File: doc/nrf/libraries/gazell/gzp.rst [hunk 1]
```diff
@@ -23,7 +23,7 @@ Requirements
 
 The Gazell Pairing library requires the same resources as the :ref:`Gazell Link Layer <ug_gzll_resources>`.
 
-In addition, the Gazell Pairing library also employs three nRF52 Series peripherals:
+In addition, the Gazell Pairing library also employs three nRF52/3 Series peripherals:
```
Comment (peknis) [inline doc/nrf/libraries/gazell/gzp.rst:26]:
```suggestion
In addition, the Gazell Pairing library also employs three nRF52 and nRF533 Series peripherals:
```

File: doc/nrf/ug_gzll.rst [hunk 2]
```diff
@@ -40,20 +40,20 @@ Configuration
...
 ******************
 
-Gazell uses a fixed set of peripheral resources in System on Chips of the nRF52 series.
+Gazell uses a fixed set of peripheral resources in the 2.4 GHz SoC.
 To ensure correct operation, Gazell requires exclusive access to the following resources:
 
 * Radio
 * Timer
-* Three PPI channels
+* Three PPI / DPPI channels
```
Comment (peknis) [inline doc/nrf/ug_gzll.rst:56]:
```suggestion
* Three PPI/DPPI channels
```

File: doc/nrf/ug_gzll.rst [hunk 3]
```diff
@@ -216,7 +216,7 @@ The most significant byte of the four bytes long base address is the first trans
 
 .. note::
 
-   The byte order in Gazell and the nRF52 Series radio peripheral are not the same.
+   The byte order in Gazell and the nRF52/3 Series radio peripheral are not the same.
```
Comment (peknis) [inline doc/nrf/ug_gzll.rst:219]:
```suggestion
   The byte order in Gazell and the nRF52 and nRF53 Series radio peripheral are not the same.
```

File: doc/nrf/ug_gzll.rst [hunk 4]
```diff
@@ -413,7 +413,7 @@ Backwards compatibility
 The Gazell Link Layer examples are not fully compatible out of the box with the legacy examples provided in the nRFgo SDK for the nRF24L IC.
 
 The default timeslot period and channel tables require adjustment, as well as some setup to emulate the Gazell roles.
-The Gazell Low Power Host role (Host role 1) is not supported in the nRF52 Series.
+The Gazell Low Power Host role (Host role 1) is not supported in the nRF52/3 Series.
```
Comment (peknis) [inline doc/nrf/ug_gzll.rst:416]:
```suggestion
The Gazell Low Power Host role (Host role 1) is not supported in the nRF52 and nRF53 Series.
```

File: doc/nrf/ug_gzll.rst [hunk 5]
```diff
@@ -423,15 +423,15 @@ The default channel tables require adjustment.
 Depending on your project, do one of the following:
 
 * Edit the :file:`gzll_params.h` file used in the nRF24L IC projects.
-* Use the :c:func:`nrf_gzll_set_channel_table()` function in the nRF52 Series projects.
+* Use the :c:func:`nrf_gzll_set_channel_table()` function in the nRF52/3 Series projects.
```
Comment (peknis) [inline doc/nrf/ug_gzll.rst:426]:
```suggestion
* Use the :c:func:`nrf_gzll_set_channel_table()` function in the nRF52 and nRF53 Series projects.
```

File: doc/nrf/ug_gzll.rst [hunk 6]
```diff
@@ -423,15 +423,15 @@ The default channel tables require adjustment.
...
 
 Timeslot periods
 ================
 
 The Gazell Link Layer supports the following minimum timeslot periods:
 
-* 600 us timeslot period, nRF52 Series Gazell Device to nRF52 Series Gazell Host.
-* 504 us timeslot period, nRF52 Series Gazell Device to nRF24L IC Gazell Host.
+* 600 us timeslot period, nRF52/3 Series Gazell Device to nRF52/3 Series Gazell Host.
+* 504 us timeslot period, nRF52/3 Series Gazell Device to nRF24L IC Gazell Host.
```
Comment (peknis) [inline doc/nrf/ug_gzll.rst:434]:
```suggestion
* 600 us timeslot period, nRF52 and nRF53 Series Gazell Device to nRF52 and nRF53 Series Gazell Host.
* 504 us timeslot period, nRF52 and nRF53 Series Gazell Device to nRF24L IC Gazell Host.
```

File: doc/nrf/ug_gzll.rst [hunk 7]
```diff
@@ -446,12 +446,12 @@ In addition, the relation between the Device and Host timing parameters should b
 Depending on your project, do one of the following:
 
 * Edit the :file:`gzll_params.h` file used in the nRF24L IC projects.
-* Use the :c:func:`nrf_gzll_set_timeslot_period()` function in the nRF52 Series projects (nRF52 Series Gazell timeslot period = 0.5*GZLL_RX_PERIOD).
+* Use the :c:func:`nrf_gzll_set_timeslot_period()` function in the nRF52/3 Series projects (nRF52/3 Series Gazell timeslot period = 0.5*GZLL_RX_PERIOD).
```
Comment (peknis) [inline doc/nrf/ug_gzll.rst:449]:
```suggestion
* Use the :c:func:`nrf_gzll_set_timeslot_period()` function in the nRF52 and nRF53 Series projects (nRF52 and nRF53 Series Gazell timeslot period = 0.5*GZLL_RX_PERIOD).
```

File: doc/nrf/ug_gzll.rst [hunk 8]
```diff
@@ -446,12 +446,12 @@ In addition, the relation between the Device and Host timing parameters should b
...
 
 Emulating legacy Gazell roles
 =============================
 
-The Gazell Link Layer protocol for the nRF52 Series is compatible with the most useful roles of the Gazell Link Layer for the nRF24L IC.
+The Gazell Link Layer protocol for the nRF52/3 Series is compatible with the most useful roles of the Gazell Link Layer for the nRF24L IC.
```
Comment (peknis) [inline doc/nrf/ug_gzll.rst:454]:
```suggestion
The Gazell Link Layer protocol for the nRF52 and nRF53 Series is compatible with the most useful roles of the Gazell Link Layer for the nRF24L IC.
```

File: doc/nrf/ug_gzll.rst [hunk 9]
```diff
@@ -204,7 +204,7 @@ Each logical address on the nodes is termed a *pipe*.
 Each pipe maps to one on-air address used when transmitting or receiving packets.
 
 The on-air addresses are composed of a 2-4 bytes long "base address" in addition to a 1-byte prefix address.
-The radio of the nRF52 Series uses an alternating sequence of 0s and 1s as the preamble of the packet.
+The 2.4 GHz proprietary radio uses an alternating sequence of 0s and 1s as the preamble of the packet.
```
Comment (peknis) [inline doc/nrf/ug_gzll.rst:207]:
```suggestion
The 2.4 GHz proprietary radio uses an alternating sequence of 0 s and 1 s as the preamble of the packet.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Comment by leewkb4567** [comment]
@peknis Your comments are addressed.

**Review by peknis — APPROVED** [review]

**Comment by carlescufi** [comment]
@leewkb4567 is this still DNM?

**Comment by leewkb4567** [comment]
@carlescufi CI adds DNM label when manifest points to the nrfxlib gzll PR for nRF5340.

Rebase to current main.

**Comment by carlescufi** [comment]
> @carlescufi CI adds DNM label when manifest points to the nrfxlib gzll PR for nRF5340.
> 
> Rebase to current main.

Oh right, I merged the nrfxlib PR now.

**Comment by leewkb4567** [comment]
Rebase to current main, which manifest has the nRF5340 GZLL library.

**Comment by leewkb4567** [comment]
Rebase to current main.

**Comment by leewkb4567** [comment]
Rebase to current main.

**Comment by leewkb4567** [comment]
Rebase to current main.

**Pull request closed without merging**